src/components/MobileHomePageBurgerActive.tsx

```diff
@@ -0,0 +1,88 @@
+import { TypeCategory, headerCategories } from '@/data/categoriesHeader';
+import React from 'react';
+import logoProduct from '@/virus.png';
+import { buildIconPath } from '@/utils/Utils';
+
+const MobileHomePageBurgerActive = () => {
+  const ID_OUR_PLATEFORME = 8;
+  const ID_ABOUT = 7;
+  return (
+    <div className="flex flex-col mt-2 ">
+      <section id="actions" className="flex flex-col items-center  space-y-2">
+        <button className="bg-vintedGreen w-10/12 py-2 text-white text-center rounded-md  ">
+          Vends tes articles
+        </button>
+        <button className="bg-white text-vintedGreen w-10/12 py-2 text-white text-center rounded-md border-vintedGreen border-2">

```

Comment:
Message de l'extension Tailwind CSS IntelliSense : 'text-vintedGreen' applies the same CSS properties as 'text-white'.(cssConflict)

Reply:
Corrigé